.github/workflows/issues-close-project-done.yaml

```diff
@@ -0,0 +1,28 @@
+name: Project Issue State Sync
+on:
+  push:
+    branches:
+      - idwa-28-close-issues-on-project-done
```

Comment:
Would this limit the GHA to a single branch?

Reply:
this is how you test actions before pushing to main

Reply:
the file is updated for main now